libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -447,13 +448,6 @@ export function Tracker(
      */
     function sendRequest(request: PayloadBuilder) {
       // Set to true if Opt-out cookie is defined
```

Comment:
This comment probably should not be here anymore as the local variable was removed.

Reply:
Looking into the code, the `getBrowserDataPlugin` is always added without any condition on the tracking side and the `beforeTrack` is always called together with the `sendRequest` function.
To declutter the duplication a bit, I took the logic and left it on just one place, which for the moment coincided with my thinking that `sendRequest` should do one thing only. But I totally agree that it is a point of improvement. I will squeez it in on this or the next touch of the file.